util/src/cryptopp_digest_creator.cpp

```diff
@@ -33,20 +33,19 @@ CryptoppDigestCreator::CryptoppDigestCreator() {
   internalState_ = p;
 }
 
-size_t CryptoppDigestCreator::digestLength() { return DigestType::DIGESTSIZE; }
+size_t CryptoppDigestCreator::digestLength() const { return DigestType::DIGESTSIZE; }
 
-bool CryptoppDigestCreator::compute(const char* input,
-                                    size_t inputLength,
-                                    char* outBufferForDigest,
-                                    size_t lengthOfBufferForDigest) {
+bool CryptoppDigestCreator::computeDigest(const char* input,
+                                          const size_t inputLength,
+                                          char* outBufferForDigest,
+                                          const size_t lengthOfBufferForDigest) {
```

Comment:
remove the const here. Why did u added this?

Reply:
`const` is added with an intention of "do not modify the input parameters".

Reply:
I think const is not needed here.

Reply:
Removed.